lib/routers/user.js

```diff
@@ -86,6 +87,7 @@ module.exports = (settings) => {
       .then(() => next());
   });
 
+  router.use('/email-preferences', emailPreferencesRouter(settings));
```

Comment:
It's generally good practice to mount `/something` routes above `/` routes, but also it doesn't matter enough to not merge if you don't want to.